Automattic/wp-calypso
# Signup reskin: Mobile tablet views

#### Changes proposed in this Pull Request

This PR is towards implementing the reskinned signup experiment. For more context on the experiment, read pbxNRc-IO-p2.

NOTE:

This PR implements changes for UI compatibility in mobile and tablet views.

Here's the [Figma file](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536) for reference.

For the record:
Parent PR: https://github.com/Automattic/wp-calypso/pull/51524 and root PR: https://github.com/Automattic/wp-calypso/pull/50512

**Screenshots of treatment**

**Mobile**
Domains step without search results - https://cloudup.com/isr5Cnyx-gC
Domains step with search results - https://cloudup.com/ibZv4p2Po4E
Plans step - https://cloudup.com/iFmHybte2o0

**iPad**
Domains step - https://cloudup.com/iCoorxmzvwE

**iPad Pro**
Domains step - https://cloudup.com/i-uS_-aBT_F



#### Testing instructions



* Assign yourself to treatment from Abacus, and visit /start.
* In mobile and tablet viewports, verify that the domain and plans steps UI matches with the [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536). Note that the user step's UI is handled separately in https://github.com/Automattic/wp-calypso/pull/51648
* Assign yourself to control in Abacus and verify that the control group experience is unaffected.
